src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private Ball ball;
+
+    public Lottery() {
+        ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        String randomColor = colorSupplier.getRandomColor();
+        ball.setColor(randomColor);
+
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
```

Comment:
the same about dependencies initialization and the magic number.
also, move this logic to the method ```getRandomBall()```